apps/docs/packages/telegram-apps-sdk-vue.md

```diff
@@ -106,41 +101,59 @@ onUnmounted(() => {
 A helper that allows you to use our [signals](./telegram-apps-signals.md) in the application. It
 returns a Vue ref which updates every time, our signal changes.
 
-```vue
-<script setup lang="ts">
-import { watchEffect, onMounted, onUnmounted } from 'vue';
-import { backButton, useSignal } from '@telegram-apps/sdk-vue';
-
-const isVisible = useSignal(backButton.isVisible);
-
-watchEffect(() => {
-  console.log('The button is', isVisible.value ? 'visible' : 'invisible');
-});
+```ts [useMainButton.vue]
+/**
+ * Composable which encapsulates mainButton interaction logic
+ */
+import { mainButton, useSignal } from '@telegram-apps/sdk-vue';
 
-onMounted(() => {
-  backButton.show();
-});
+export function useMainButton() {
+  if (!mainButton.isMounted()) {
+    mainButton.mount();
+  }
 
-onUnmounted(() => {
-  backButton.hide();
-});
-</script>
+  const isVisible = useSignal(mainButton.isVisible);
 
-<template></template>
+  return { isVisible };
+}
 ```
 
 ### `useLaunchParams`
 
-A function that returns the mini application's launch parameters.
+A function that returns the mini application's launch parameters. For Vue.js it's just `retrieveLaunchParams` function from `@telegram-apps/sdk`.
 
-```vue
-<script setup lang="ts">
+```ts [useInitApp.ts]
+/**
+ * Composable which encapsulates start param processing logic
+ */
 import { useLaunchParams } from '@telegram-apps/sdk-vue';
 
-const lp = useLaunchParams();
-</script>
+export function useInitApp() {
```

Comment:
What does the function do? It returns `undefined` anyway

Reply:
It has a side-effect. That particular example shows how app may implement startParam processing using `useLaunchParams`